apps/authentication-service/cypress/e2e/clients.cy.ts

```diff
@@ -0,0 +1,167 @@
+import { Client } from "auth0";
+
+const USER_ID: string = "auth0|667c3d0c90b963e3671f411e";
+
+describe("Client API", () => {
+  const PORT: number = (Cypress.env("PORT") as number) || 8089;
+  let accessToken: string;
+  let m2mToken: string;
+  let newClientId: string;
+  let newClientSecret: string;
+
+  const getTokens = () => {
+    cy.window()
+      .its("localStorage")
+      .invoke("getItem", "jwt-m2m")
+      .then((token: string) => {
+        if (token) {
+          m2mToken = token;
+        }
+      });
+    cy.window()
+      .its("localStorage")
+      .invoke("getItem", "jwt-client")
+      .then((token: string) => {
+        if (token) {
+          accessToken = token;
+        }
+      });
+  };
+
+  before(() => {
+    getTokens();
+    if (!accessToken) {
+      cy.loginM2MAuth0();
+    }
+    if (!m2mToken) {
+      cy.loginClientAuth0();
+    }
+    getTokens();
+  });
+
+  it("creates a client without access token", () => {
+    cy.request({
+      failOnStatusCode: false,
+      method: "POST",
+      url: `http://localhost:${PORT}/v1/clients`,
+      headers: {
+        ContentType: "application/json",
+      },
+    }).then((response: Cypress.Response<{ body: Client }>) => {
+      expect(response.status).to.eq(401);
+      expect(response.body)
+        .property("message")
+        .to.eq("Requires Authentication");
+    });
+  });
+
+  it("clears the client_id from user metadata, tries creating a client without access token, creates a client, fails if another client request is made, gets the newly created client, and deletes the client", () => {
+    // 1. Delete the Client
+    // 2. Create a new Client
+    // 3. Get the client
+    // 4. Try to create again, make sure error is returned
+    // 5. Delete the Client
+    // 6. Get the client, to make sure it's deleted
+
+    cy.request({
+      failOnStatusCode: false,
+      method: "DELETE",
+      url: `http://localhost:${PORT}/v1/clients`,
+      headers: {
+        Authorization: `Bearer ${accessToken}`,
+      },
+    }).then((response) => {
```

Comment:
This test will fail if there is a client still attached. I wouldn't check the response at all. This is just there to try and make sure everything is ready to run the test.